components/webfield/ProgramChairConsole.js

```diff
@@ -67,6 +67,8 @@ const ProgramChairConsole = ({ appContext }) => {
     customStageInvitations,
     assignmentUrls,
     emailReplyTo,
+    reviewerEmailFuncs,
```

Comment:
will ARR have multiple reviewer groups too? such as technical reviewers, senior reviewers...
maybe reviewerEmailFuncs needs to be an object which contains multiple reviewer groups
so that we don't need to add more config vars when reviewerName is changed to array

Reply:
For now ARR only has 1 reviewer group, but I believe there is some interest in secondary reviewers to train newer reviewers.

For your suggestion, then maybe `reviewerEmailFuncs` should look something like:
```
{
  Reviewers: [
    {
      label: 'Reviewer Two', filterFunc: `
      return row.reviewerProfileId === '~Reviewer_ARRTwo1'
      `
    },
  ],
  Secondary_Reviewers: [
  {
    label: 'Secondary Reviewer Two', filterFunc: `
    return row.reviewerProfileId === '~SecondaryReviewer_ARRTwo1'
    `
  }
  ]
}
```

Reply:
I would prefer if we could keep this PR more focused and implement the secondary reviewer changes separately when we know all of ARR's requirements.